lib/ex_aws/sts/parsers.ex

```diff
@@ -36,6 +36,32 @@ if Code.ensure_loaded?(SweetXml) do
       {:ok, Map.put(resp, :body, parsed_body)}
     end
 
+    def parse({:ok, %{body: xml} = resp}, :get_access_key_info) do
+      parsed_body =
+        SweetXml.xpath(xml, ~x"//GetAccessKeyInfoResponse",
+          account: ~x"./GetAccessKeyInfoResult/Account/text()"s,
+          request_id: request_id_xpath()
+        )
+
+      {:ok, Map.put(resp, :body, parsed_body)}
+    end
+
+    def parse({:ok, %{body: xml} = resp}, :assume_role_with_s_a_m_l) do
+      parsed_body =
+        xml
+        |> SweetXml.xpath(~x"//AssumeRoleWithSAMLResponse",
+          access_key_id: ~x"./AssumeRoleWithSAMLResult/Credentials/AccessKeyId/text()"s,
```

Comment:
Good find, I went ahead and added basic tests for all the parser function clauses using the sample data from AWS. I *hope* the sample responses are all good from AWS here, because I updated the paths to match the documentation (not always the case). 

Random aside, what are your opinions on setting up something like TravisCI or Circle?

Reply:
Nice, thanks!

Circle would be great. I'll see if I can set that up on the weekend.